10up/ElasticPress
# Fix: Facet returns no result for a term having accent characters

### Description of the Change

This PR fixes the issue where the facet returns no result for a term having accent characters


Closes #3023 

### How to test the Change


- Create a new category `غير-مصنف`
- Assign some posts to new category
- Go to frontend and select `غير-مصنف` from taxonomy widget
- EP should returns the from `غير-مصنف` category

### Changelog Entry

> Fixed - Facet returns no result for a term having accent characters


### Credits

Props @burhandodhy 


### Checklist:
